modules/api/src/main/java/org/apache/ignite/table/manager/IgniteTables.java

```diff
@@ -101,7 +73,8 @@
     void alterTable(String name, Consumer<TableChange> tableChange);
 
     /**
-     * Alter a cluster table.
+     * Alter a cluster table. If appropriate table does not be found, a future will be
```

Comment:
```suggestion
     * Alter a cluster table. If appropriate table cannot be found, a future will
```

Reply:
The actual phrase is "If an appropriate table does not exist..."